packages/models-library/tests/test_service_settings_labels.py

```diff
@@ -387,3 +392,183 @@ def test_not_allowed_in_both_permit_list_and_outgoing_internet():
         f"Not allowed common_containers={{'{container_name}'}} detected"
         in f"{exec_info.value}"
     )
+
+
+@pytest.fixture
```

Comment:
why not keep all these fixtures and tests for next PR since they are skipped now. It will be simpler to review

Reply:
Some reasons are:
- TDD: i prefer to have tests ready before the features are implemented
- They are also illustrative of user-cases for follow-up PR
- I do not forget details (in case I cannot do the follow-up right away)